app/components/UI/Carousel/StackCardEmpty/StackCardEmpty.tsx

```diff
@@ -12,6 +13,9 @@ import { ANIMATION_TIMINGS } from '../animations/animationTimings';
 import { StackCardEmptyProps } from './StackCardEmpty.types';
 import { strings } from '../../../../../locales/i18n';
 
+// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires, import/no-commonjs
+const CarouselConfetti = require('../../../../animations/Carousel_Confetti.riv');
```

Comment:
non-blocking: Why are we using common js requre method here and not ESM import? We are importing other riv animations using [esm imports](https://github.com/MetaMask/metamask-mobile/blob/bafe7e1e7cc2892cf7c3a8cb92a773ea2c193ff2/app/components/Views/Onboarding/FoxAnimation.tsx#L7)

```suggestion
import CarouselConfetti from '../../../../animations/Carousel_Confetti.riv';
```